tasks/2017/submit-a-pull-request.md

```diff
@@ -19,7 +19,7 @@ Why?
 * Finally, if you do make a mistake, you'll be able to learn to [fix your commits](http://zulip.readthedocs.io/en/latest/git/fixing-commits.html).
 
 
-The directions for the task is as follows:
+The directions for the task is as follows (you should use Git in **command line** for this task):
```

Comment:
command line interface might be better? Also note which CLI for which OS (Mac is Terminal, for example)

Reply:
This is misleading: how does one fork the repository (the very first step) from the terminal? Are we expecting students to use the `hub` command line tool?